web-ui/src/main/resources/catalog/js/CatController.js

```diff
@@ -294,7 +294,7 @@
 
         // Retrieve main search information
         var searchInfo = userLogin.then(function(value) {
-          var url = 'qi?_content_type=json&summaryOnly=true';
+          var url = 'qi?_content_type=json&summaryOnly=true&_retired=false';
```

Comment:
doing this will alter general catalog stats in the admin page. Maybe not a good idea ?